docker-compose/wildfly-manager/docker-compose.yml

```diff
@@ -0,0 +1,246 @@
+      - xpack.security.enabled=false
+#    ports:
+#      - '${ELASTICSEARCH_PORT}:${ELASTICSEARCH_PORT}'
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+      nproc:
+        soft: 4096
+        hard: 4096
+    volumes:
+      - type: volume
+        source: elasticsearch_data
+        target: /usr/share/elasticsearch/data
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "10"
+        mode: non-blocking
+    #    labels:
+    #      - traefik.http.routers.elasticsearch-router.tls=true
+    #      - traefik.http.routers.elasticsearch-router.entrypoints=https
+    #      - traefik.http.routers.elasticsearch-router.rule=PathPrefix(`/elasticsearch`)
+    #      - traefik.http.routers.elasticsearch-router.service=elasticsearch-service
+    #      - traefik.http.services.elasticsearch-service.loadbalancer.server.port=9200
+    restart: always
+
+  keycloak:
+    image: 'jboss/keycloak:15.0.0'
+    container_name: keycloak
+    environment:
+      - "PROXY_ADDRESS_FORWARDING=true"
+      - 'DB_VENDOR=POSTGRES'
+      - 'DB_ADDR=postgres'
+      - 'DB_DATABASE=${POSTGRES_KEYCLOAK_DATABASE}'
+      - 'DB_USER=${POSTGRES_KEYCLOAK_USER}'
+      - 'DB_PASSWORD=${POSTGRES_KEYCLOAK_PASSWORD}'
+      - 'DB_SCHEMA=public'
+      - 'KEYCLOAK_USER=${KEYCLOAK_ADMIN_USERNAME}'
+      - 'KEYCLOAK_PASSWORD=${KEYCLOAK_ADMIN_PASSWORD}'
+      - 'KEYCLOAK_IMPORT=/tmp/keycloak-default-realm.json'
+      - 'JDBC_PARAMS="useSSL=false"' # No need for SSL between Postgres <-> Keycloak in a docker network
+#    ports:
+#      - '8445:8080'
+    depends_on:
+      - postgres
+    volumes:
+      - type: bind
+        source: $PWD/../common/keycloak-default-realm.json
+        target: /tmp/keycloak-default-realm.json
+      - type: volume
+        source: keycloak_logs
+        target: /opt/jboss/keycloak/standalone/log
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "10"
+        mode: non-blocking
+    labels:
+#      - traefik.http.routers.keycloak-router.tls=true
+      - traefik.http.routers.keycloak-router.entrypoints=http
+      - traefik.http.routers.keycloak-router.rule=PathPrefix(`/auth`)
+      - traefik.http.routers.keycloak-router.service=keycloak-service
+      - traefik.http.services.keycloak-service.loadbalancer.server.port=8080
+    restart: always
+
+  ui:
+    image: 'apiman/wildfly-manager:latest'
+    container_name: ui
+    environment:
+      - "APIMAN_AUTH_URL=${KEYCLOAK_URL}"
+      - "APIMAN_AUTH_REALM=${KEYCLOAK_REALM}"
+      - "APIMAN_AUTH_APIMAN_SECRET=${KEYCLOAK_APIMAN_SECRET}"
+      - "APIMAN_AUTH_APIMANUI_SECRET=${KEYCLOAK_APIMANUI_SECRET}"
+      - "ALLOW_ANY_HOST=${SELF_SIGNED}"
+#    ports:
+#      - '8443:8080'
+    depends_on:
+      - postgres
+      - keycloak
+    volumes:
+      # uncomment if you want to use your own configuration files
```

Comment:
I would put that into the docs.
It's easier if we do not spread examples all over the files.
Will be also hard to update for us at some point.